src/main/java/gov/nasa/pds/api/engineering/controllers/MyProductsApiBareController.java

```diff
@@ -93,6 +94,36 @@ public MyProductsApiBareController(ObjectMapper objectMapper, HttpServletRequest
 	        
     }
     
+    protected Products getLIDVIDs(List<String> lidvids, List<String> fields, HashSet<String> uniqueFields, boolean onlySummary) throws IOException
```

Comment:
I also would like this function to be moved in the business package (gov/nasa/pds/api/engineering/elasticsearch/business) since it returns the API model object.

The following method 'getProducts' should also be in this package actually. 

This comment is not blocking for the merge of the pull request.